lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -160,7 +160,7 @@ def test_connection_error(self, monkeypatch, agent_logger):
                 self.config,
                 agent_logger,
             )
-            crt.copy_result_tb("token")
+            res = crt.copy_result_tb("token")
```

Comment:
You can (and probably should) omit the assignment here, since it should never actually happen (and since we do not reference the variable).

Ditto line 190.